src/test/java/com/owncloud/android/lib/common/operations/RemoteOperationResultTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.owncloud.android.lib.common.operations
```

Comment:
Licence please :)
There is even an automated way with templates within AS, if you like such things :)

Reply:
Fixed, I have the template setup for Files but forgot for the library :(